### PR TITLE
BAU: Bump Go yaml package to 2.2.8

### PR DESCRIPTION
**Pull request by katstevens**
For some reason Dependabot was struggling to create this update automatically. I also couldn't get the tests to pass unless I moved the dependency to the main `require` section rather than the indirect section.

### CODE REVIEW

File: integration/integration_test.go [hunk 1]
```diff
@@ -19,23 +19,23 @@ func TestIntegration(t *testing.T) {
...
+		buildCmd := exec.Command("docker compose", "build")
 		session, err := gexec.Start(buildCmd, GinkgoWriter, GinkgoWriter)
 		Expect(err).ShouldNot(HaveOccurred())
 		Eventually(session, 60).Should(gexec.Exit(0))
 	})
 
 	BeforeEach(func() {
 		upCmd := exec.Command(
-			"docker-compose", "up", "--detach", "--force-recreate",
+			"docker compose", "up", "--detach", "--force-recreate",
```
Comment (jfharden) [inline integration/integration_test.go:30]:
```suggestion
			"docker", "compose", "up", "--detach", "--force-recreate",
```

Comment (jfharden) [reply integration/integration_test.go:30]: This change should work, with it in a single string it's looking for an executable with a space in the file name

**Review by jfharden — COMMENTED** [review]
Just a suggestion on how to fix the docker compose issue

File: integration/integration_test.go (hunk 1 above)
Comment (katstevens) [reply integration/integration_test.go:30]: Ah fabulous - I knew it would be something tiny like this! Thanks @jfharden, I'll give it a whirl

**Review by jfharden — APPROVED** [review]

File: integration/integration_test.go (hunk 1 above)
Comment (jfharden) [reply integration/integration_test.go]: aaaaand a green build 🥳

**Pull request merged**